salt/states/grains.py

```diff
@@ -144,17 +154,6 @@ def list_present(name, value, delimiter=DEFAULT_TARGET_DELIM):
             if set(value).issubset(set(__salt__['grains.get'](name))):
                 ret['comment'] = 'Value {1} is already in grain {0}'.format(name, value)
                 return ret
-            elif name in __context__.get('pending_grains', {}):
```

Comment:
My bad. When I had to roll back my branch and re-apply the logic for this change I did not look at the diff closely enough. Do you want to cancel the PR and for me to submit a new one or can you resolve this your side?